.github/workflows/extract-translations.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  sync-translations:
+    name: Extract translations and submit a PR
+    runs-on: ubuntu-latest
+    env:
+      BRANCH: ${{ github.ref_name }}
+    steps:
+      - name: Checkout module-translation-tool
+        uses: actions/checkout@v4
+        with:
+          repository: PrestaShopCorp/module-translation-tool
```

Comment:
@jolelievre need your insights here. Is that okay as-is or do we have to move this repository?

Reply:
I didn't even know about this repo 😅 We just need to check if there are no sensitive data in the repo (like old keys/secrets) And then I guess it could be moved to the open source organisation

But if it's not blocking your CI right now I think you can start by using this address and update it when the repo will be moved